apps/projects/app/components/Content/Settings.js

```diff
@@ -5,16 +5,19 @@ import NumberFormat from 'react-number-format'
 
 const bountyDeadlines = ['Weeks', 'Days', 'Hours']
 const bountyDeadlinesMul = [168, 24, 1] // it is one variable in contract, so number * multiplier = hours
-const bountyCurrencies = ['BTC', 'ETH', 'TL', 'ANT', '🦄']
 
 class Settings extends React.Component {
-  state = {}
+  state = {
+    bountyCurrencies: this.props.tokens.map (token => token.symbol)
```

Comment:
This fails with TypeError: r.props.tokens.map is not a function message on my system.

Reply:
We should not set the state that way, why not use the props directly? Is it really needed to extra cache the immutable props into the state? It is a react anti-pattern and should be refactored